### PR TITLE
add support for textarea in isElementTabbable method

### PR DESCRIPTION
**Pull request by ipip2005**
Fix the focusZone/focusTrapZone doens't support textarea tag.

### CODE REVIEW

File: src/utilities/focus.ts [hunk 1]
```diff
@@ -130,6 +130,7 @@ export function isElementTabbable(element: HTMLElement): boolean {
     (element.tagName === 'A' ||
       (element.tagName === 'BUTTON' && !(element as HTMLButtonElement).disabled) ||
```
Comment (cschlechty) [inline src/utilities/focus.ts]: Not specific to your change, but why do we need to cast all these as button? @dzearing

**Comment by dzearing** [comment]
Approved

File: src/utilities/focus.ts (hunk 1 above)
Comment (dzearing) [inline src/utilities/focus.ts]: because HTMLElement doesn't have a disabled property

Comment (cschlechty) [inline src/utilities/focus.ts]: I guess I was thinking more of the C# style 'as' operator where it actually checks they object type and returns null if it doesn't match, where this just swaps the interface.

Comment (dzearing) [inline src/utilities/focus.ts]: That doesn't work in typescript. :) But the static analysis works, which is why we need to cast so it stops complaining.

File: src/utilities/focus.ts [hunk 2]
```diff
@@ -130,6 +130,7 @@ export function isElementTabbable(element: HTMLElement): boolean {
...
       (element.tagName === 'INPUT' && !(element as HTMLButtonElement).disabled) ||
+      (element.tagName === 'TEXTAREA' && !(element as HTMLButtonElement).disabled) ||
```
Comment (dzearing) [inline src/utilities/focus.ts]: Shouldn't the type be something like HTMLTextAreaElement for correctness?

**Comment by ipip2005** [comment]
@dzearing I modified the typing and it seems ok to be merged, could you trigger a merge for this?

**Pull request merged**